src/app/public/modules/radio/radio.component.ts

```diff
@@ -97,9 +93,9 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
    */
   @Input()
   public set disabled(value: boolean) {
-    const newDisabledState = SkyFormsUtility.coerceBooleanProperty(value);
```

Comment:
Why did we remove the call to `coerceBooleanProperty` ? This allowed a consumer to provide both a boolean or a string to the `disabled` attribute. See more here: https://netbasal.com/trust-but-verify-coerce-your-component-inputs-bdb743e8b579

Reply:
My mistake, definitely should have this in our `@Input()`s. Thanks for sending that article! Fixed this, and another one in checkbox.component.ts as well, in last commit